serde-sarif/Cargo.toml

```diff
@@ -22,6 +22,7 @@ unexpected_cfgs = { level = "warn", check-cfg = ['cfg(doc_cfg)'] }
 [features]
 default = []
 clippy-converters = ["cargo_metadata", "regex", "anyhow"]
+miri-converters = ["clippy-converters"]
```

Comment:
i think it's probably better to just explicity state the same features here (rather than point to clippy's)